planet.rb

```diff
@@ -0,0 +1,18 @@
+
+require 'pry'
+
+class Planet
+  def initialize(name, color, mass_kg, distance_from_sun_km, fun_fact)
+    @name = name
+    @color = color
+    @mass_kg = mass_kg
+    @distance_from_sun_km = distance_from_sun_km
+    @fun_fact = fun_fact
+  end
+  attr_reader :name, :color, :mass_kg, :distance_from_sun_km, :fun_fact
```

Comment:
By convention this line should appear above `initialize`.  Ruby doesn't care, but it's typically where we place it for readability.